servo-core/src/main/java/com/netflix/servo/util/Preconditions.java

```diff
@@ -39,13 +39,19 @@ private Preconditions() {
    * Ensures the truth of an expression involving one or more parameters to the
    * calling method.
    *
+   * @param errorMessage the error message - can be a <a href="../util/Formatter.html#syntax">format string</a>
```

Comment:
I don't think this link will work, it is relative to the javadocs for servo instead of pointing to the formatter in the jdk.

https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html#syntax